be/src/olap/txn_manager.h

```diff
@@ -48,6 +48,22 @@
 
 namespace doris {
 
+struct TabletTxnInfo {
+    PUniqueId load_id;
+    RowsetSharedPtr rowset;
+    int64_t creation_time;
+
+    TabletTxnInfo(
+        PUniqueId load_id,
+        RowsetSharedPtr rowset) :
+        load_id(load_id),
+        rowset(rowset) {
+        creation_time = time(NULL);
```

Comment:
the name is the same:load_id(load_id)? IS there no problems?
and creation_time can also be put in initialization list,
create_time(time(nullptr))